pkgs/applications/misc/rofi/default.nix

```diff
@@ -55,13 +63,16 @@ stdenv.mkDerivation rec {
     xcbutilxrm
     xcb-util-cursor
     which
+  ] ++ lib.optionals waylandSupport [
+    wayland-protocols
+    wayland
   ];
 
   doCheck = false;
 
   meta = with lib; {
     description = "Window switcher, run dialog and dmenu replacement";
-    homepage = "https://github.com/davatorium/rofi";
+    homepage = "https://github.com/${owner}/rofi";
```

Comment:
Not a fan of such hacks because no I no longer can click on the link. Maybe we split it into two files or use overrideAttr.